language_sentiment_text.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "language_sentiment_text")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-language
+
+# sample-metadata
+#   title: Analyzing Sentiment
+#   description: Analyzing Sentiment in a String
+#   usage: python3 samples/v1/language_sentiment_text.py [--text_content "I am so happy and joyful."]
+
+# [START language_sentiment_text]
+from google.cloud import language_v1
```

Comment:
`from google.cloud import language_v1` --> `from google.cloud import language_v1 as language` 
Makes it a lot easier to perform updates